airflow/www_rbac/static/js/clock.js

```diff
@@ -18,24 +18,25 @@
  */
 require('./jqClock.min');
 
-$(document).ready(function () {
-  x = new Date();
+$(document).ready(() => {
```

Comment:
This is the es6 syntax, and we should make sure that the compiled result is in es5, as not all browsers support es6. (especially IE)

Reply:
Good point.  What are your thoughts on bringing in a babel build step to transpile the ES6 to ES5 vs writing as ES5?  Either as part of this issue or separately a follow up.

Reply:
Please note that most of the custom JS is written inline in .html files, and we are not yet considering that javascript in webpack, that means, we won't be able to transpile that javascript to ES5. (which is fine for now)
I am working on another issue to extract all inline JS from html files to separate .js files. 
https://issues.apache.org/jira/browse/AIRFLOW-2804

My suggestion would be to implement the ES6->ES5 tranpilation as part of this issue. And once this PR gets merged, we'll be able to extract all inline JS into separate .js files. 
We already have a JIRA issue for that: https://issues.apache.org/jira/browse/AIRFLOW-2730

Reply:
Sounds good.  I will stick with the ES5 for now for this PR.